tests/unit/backend/corpora/dataset_processing/test_h5ad_data_file.py

```diff
@@ -174,6 +174,28 @@ def test__to_cxg__with_sparse_column_encoding(self):
         # Clean up
         remove(sparse_with_column_shift_filename)
 
+    def test__slash_in_attribute_name(self):
```

Comment:
This bug repro code lifted from the [TileDB-Py bug that Bruce posted](https://github.com/TileDB-Inc/TileDB-Py/issues/294)

Using a known-bad version of tiledb (e.g. 0.8.0) line 194 will throw a `TileDBError` and cause the test to fail.